README-WAF.md

```diff
@@ -62,7 +62,18 @@ waf list
 
 ## Program groups ##
 
-The programs in ardupilot are categorized into the following groups:
+Program groups are used to represent a class of programs. They can be used to
+build all programs of a certain class without having to specify each program. A
+program can belong to more than one group.
```

Comment:
What isn't clear for me is the "_A program can belong to more than one group._" sentence. I guess what you mean here is that it can belong to Main groups and Main product groups? When I read it what I though was that a program could belong to bin and tests, for example.